tabpfn_client/client.py

```diff
@@ -157,15 +158,20 @@ class ServiceClient(Singleton):
     httpx_timeout_s = (
         4 * 5 * 60 + 15  # temporary workaround for slow computation on server side
     )
-    httpx_client = httpx.Client(
+    httpx_client = AnalyticsHttpClient(
```

Comment:
Why is this not in the constructor?

Reply:
This singleton class doesn't have a constructor.